src/components/dashboard/widgets/dw_engagement_count.tsx

```diff
@@ -23,44 +24,62 @@ import {
   EngagementStatus,
   getEngagementStatus,
 } from '../../../schemas/engagement';
+import { useHistory } from 'react-router';
 import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
 
+
 export interface EngagementCountWidgetProps {
   engagements?: Partial<Engagement>[];
 }
 interface CountComponentProps {
   count: number;
-  title: string;
-  subtitle: string;
   icon: ComponentClass<SVGIconProps>;
+  status: string;
+  subtitle: string;
+  title: string;
+  url: string;
 }
-const CountComponent = ({ icon: Component, ...props }: CountComponentProps) => (
-  <Flex
-    direction={{ default: 'column' }}
-    alignItems={{ default: 'alignItemsCenter' }}
-    justifyContent={{ default: 'justifyContentCenter' }}
-    style={{ height: '100%' }}
-  >
-    <FlexItem flex={{ default: 'flex_1' }}>
-      <Text component={TextVariants.h4}>{props.title}</Text>
-    </FlexItem>
-    <FlexItem>
-      <Flex
-        direction={{ default: 'column' }}
-        justifyContent={{ default: 'justifyContentFlexEnd' }}
-        alignItems={{ default: 'alignItemsCenter' }}
-      >
-        <Component size="xl" />
-        <TextContent>
-          <Text component={TextVariants.h1}>{props.count}</Text>
-          <Text style={{ fontStyle: 'italic', fontSize: '0.8em' }}>
-            {props.subtitle}
+const CountComponent = ({ icon: Component, ...props }: CountComponentProps) => {
+  const history = useHistory();
+  return(
+    <Flex
+      direction={{ default: 'column' }}
+      alignItems={{ default: 'alignItemsCenter' }}
+      justifyContent={{ default: 'justifyContentCenter' }}
+      style={{ height: '100%' }}
+    >
+      <FlexItem flex={{ default: 'flex_1' }}>
+        <Button
+          variant="link"
+          onClick={() => {
+            history.push(props.url);
+          }}
+          data-cy={`button_${props.status}`}
+        >
```

Comment:
I basically copied this from the old dashboard component, but used a "status" prop instead of the first letter of the title.